pkg/messagebroker/messagebroker.go

```diff
@@ -16,6 +16,9 @@ type Admin interface {
 
 	// deletes an existing topic
 	DeleteTopic(context.Context, DeleteTopicRequest) (DeleteTopicResponse, error)
+
+	// adds partitions to an existing topic
+	AddTopicPartitions(context.Context, AddTopicPartitionRequest) (*AddTopicPartitionResponse, error)
```

Comment:
is this only add or we can use it to remove as well ?

Reply:
Kafka does not support decreasing partition counts on an existing topic.